Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -287,8 +287,11 @@ endif()
 find_path(SwiftShims_INCLUDE_DIR "shims/module.modulemap" HINTS
   ${SwiftShims_INCLUDE_DIR_HINTS})
 add_library(swiftShims INTERFACE IMPORTED GLOBAL)
-set_target_properties(swiftShims PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${SwiftShims_INCLUDE_DIR}/shims")
+target_include_directories(swiftShims INTERFACE
+  "${SwiftShims_INCLUDE_DIR}/.."
```

Comment:
I might've missed the reason for this path. Can you please add a comment explaining why this is added?